terraform/main.tf

```diff
@@ -0,0 +1 @@
+resource "null_resource" "foo" {}
```

Comment:
Do we need this file for `create` ?

Reply:
no, feel free to delete the `terraform` directory, its from testing